docs/developer-docs-home.md

```diff
@@ -2,31 +2,61 @@
-    This document contains a comprehensive list of available APIs that your field plug-in can use to interact with the form.  
-* [Using field plug-ins (user documentation)](https://docs.surveycto.com/02-designing-forms/03-advanced-topics/06.using-field-plug-ins.html)  
-    This article is written for form designers and will explain how to install and use field plug-ins in SurveyCTO.
+A field plug-in can be thought of as a web page. When filling out a SurveyCTO form, this web page will load instead of the standard field view. A field plug-in effectively 'takes over' the field on which it is used. In this way, field plug-ins enable you to have complete control over the appearance and functionality of questions in your data collection instrument.
+
+If you've ever created your own web page using HTML, CSS, and Javascript, then you already understand the underlying technology. A field plug-in uses those three basic web technologies to create a web view whenever the field is loaded in a form. You can use all the features of standard Javascript to control what happens in the field plug-in itself, then use our form APIs to control how your field plug-in interacts with the rest of the form.
+
+## What can a field plug-in do
+
+The short answer is that a field plug-in can do anything that a normal web page can do. When filling out a SurveyCTO form and you arrive at a field that uses a field plug-in, a web page is created using your field plug-in's code. On Android, a field plug-in is loaded in its own fully-functional Android WebView. On iOS, it is loaded in a WKWebView. In web forms, it is loaded in an iframe. No matter which platform is used, a field plug-in is free to use HTML, CSS, and Javascript in the same way any normal web page can.
+
+## How can a field plug-in interact with a form
+
+Field plug-ins are loaded in their own isolated web views, but there are several ways in which your field plug-in can interact with the rest of the form and the Collect interface.
+
+* **Form APIs**  
+    We've created APIs that you can use to interact with the form in specific ways. Some of these APIs are required, and should be included in every field plug-in. Some are provided to you for convenience, and may not be applicable in every field. The full list is available in the [form API reference](https://github.com/surveycto/field-plug-in-resources/blob/master/docs/api-reference.md) documentation.
+
+* **Parameters**  
+    When you configure a field to use a field plug-in, you can supply optional parameters as well. These parameters can contain expressions and field references, which get evaluated right before the field plug-in loads. This can be used to pass information or configuration settings from other parts of the form to your field plug-in. More information is avaiable within the [form API reference](https://github.com/surveycto/field-plug-in-resources/blob/master/docs/api-reference.md) documentation.
```

Comment:
```suggestion
    When you configure a field to use a field plug-in, you can supply optional parameters as well. These parameters can contain expressions and field references, which get evaluated right before the field plug-in loads. This can be used to pass information or configuration settings from other parts of the form to your field plug-in. More information is available within the [form API reference](https://github.com/surveycto/field-plug-in-resources/blob/master/docs/api-reference.md) documentation.
```